app/models/ext_management_system.rb

```diff
@@ -511,4 +511,8 @@ def self.blacklisted_events
   def self.datastore?
     IsoDatastore.where(:ems_id => all).exists?
   end
+
+  def tenant_identity
+    [User.super_admin, tenant.default_miq_group, tenant]
```

Comment:
This triad is screaming at me to be an object/struct.  cc @kbrock @matthewd @jrafanie